cli/src/pcluster/aws/ec2.py

```diff
@@ -97,28 +97,35 @@ def describe_images(self, ami_ids, filters, owners):
             return [ImageInfo(image) for image in result.get("Images")]
         raise ImageNotFoundError(function_name="describe_images")
 
-    def image_exists(self, image_name):
+    def image_exists(self, image_name: str, build_status_avaliable: bool = False):
         """Return a boolean describing whether or not an image with the given search criteria exists."""
         try:
-            self.describe_image_by_name_tag(image_name)
+            self.describe_image_by_name_tag(image_name, build_status_avaliable)
             return True
         except ImageNotFoundError:
             return False
 
     @AWSExceptionHandler.handle_client_exception
-    def describe_image_by_name_tag(self, image_name):
+    def describe_image_by_name_tag(self, image_name: str, build_status_avaliable: bool = False):
```

Comment:
I am concern if we don't pass `build_status_avaliable=true` in image property of imagebuilder.py. Because describe-image and list-image depend on that property to return ImageBuilderInfo. I notice we should delete image even if it doesn't have build_status_avaliable tag, one possible solution is to add a boolean is_available in ImageInfo, and use this boolean to determine image has build_status_avaliable or not.

Reply:
list-image fixed taking into account the new `build_status` tag